documentation/app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "license": "MIT",
   "devDependencies": {
-    "@fastly/js-compute": "^1",
+    "@fastly/js-compute": "^3",
```

Comment:
Unfortunately this broke the documentation application at https://js-compute-reference-docs.edgecompute.app/.

Posting a new release now to revert.